site/content/docs/5.2/components/dropdowns.md

```diff
@@ -774,6 +804,12 @@ Put a form within a dropdown menu, or make it into a dropdown menu, and use [mar
 
 Use `data-bs-offset` or `data-bs-reference` to change the location of the dropdown.
```

Comment:
Just leaving it here but `data-bs-offset="0,0"` is totally correct

Reply:
OK dropped it. Added a comment in the other PR